mlflow/server/js/src/components/ExperimentListView.js

```diff
@@ -44,13 +51,25 @@ export class ExperimentListView extends Component {
                title="Hide experiment list"
                className="collapser fa fa-chevron-left login-icon"/>
           </div>
+          <input
+            className="experiment-list-search-input"
+            type="text"
+            placeholder="Search Experiments"
+            value={searchInput}
+            onChange={this.handleSearchInput}
+          />
           <div className="experiment-list-container" style={{ height: experimentListHeight }}>
-            {this.props.experiments.map((e, idx) => {
+            {
+            // filter experiments based on searchInput
+            this.props.experiments.filter((e) => {
+              return e.getName().toLowerCase().includes(searchInput.toLowerCase());
+            }).map((e, idx) => {
               let active;
+              const parsedExperimentId = parseInt(e.getExperimentId(), 10);
               if (this.props.activeExperimentId) {
-                active = parseInt(e.getExperimentId(), 10) === this.props.activeExperimentId;
+                active = parsedExperimentId === this.props.activeExperimentId;
               } else {
-                active = idx === 0;
+                active = parsedExperimentId === 0;
```

Comment:
This change the default active experiment from the 1st experiment to the experiment with Id=0. Is this what we wanted?

Reply:
Good point. One issue that I encountered was that when selecting the experiment with ID 0 & entering a search term, the first item in the filtered list would be shown as active. Since `0` is a falsy value, the active mark was set where `idx === 0`.
I updated the condition in the if clause to `this.props.activeExperimentId !== undefined` and undid the change that you mentioned.